src/wrapped/wrappedlibc_private.h

```diff
@@ -1324,7 +1324,11 @@ GOWD(modfl, DFDp, modf)
 //GOW(modify_ldt, 
 //GOW(moncontrol, 
 //GO(__monstartup, 
-//GOW(monstartup, 
+#ifdef STATICBUILD
+//GO(monstartup, iFpp)
+#else
+GO(monstartup, iFpp)
```

Comment:
The doc I found state this:
```C
       void
       monstartup(u_long lowpc,	u_long highpc);
```
so, signature should be `vFLL` instead. Were did you found `iFpp`?

Reply:
```
int monstartup(caddr_t LowProgramCounter, caddr_t HighProgramCounter);
```
https://www.ibm.com/docs/zh-tw/aix/7.1?topic=m-monstartup-subroutine

And `caddr_t`:
```
typedef char *__caddr_t;
typedef __caddr_t caddr_t;
```

Thanks,
Leslie Zhai

Reply:
@ptitSeb  you are correct:

```
/usr/include/loongarch64-linux-gnu/sys/gmon.h:194:extern void monstartup (unsigned long __lowpc, unsigned long __highpc) __THROW;
```

Thanks,
Leslie Zhai